src/util/DbrHelper.sol

```diff
@@ -0,0 +1,335 @@
+pragma solidity ^0.8.20;
+
+import "src/interfaces/IERC20.sol";
+import "src/interfaces/IMarket.sol";
+import {ReentrancyGuard} from "openzeppelin-contracts/contracts/utils/ReentrancyGuard.sol";
+import {Ownable} from "openzeppelin-contracts/contracts/access/Ownable.sol";
+
+interface ICurvePool {
+    function exchange(
+        uint i,
+        uint j,
+        uint dx,
+        uint min_dy,
+        bool use_eth,
+        address receiver
+    ) external payable returns (uint);
+}
+
+interface IINVEscrow {
+    function claimDBRTo(address to) external;
+
+    function claimable() external returns (uint);
+}
+
+/// @title DbrHelper
+/// @notice Helper contract to claim DBR, sell it for DOLA and optionally repay debt or sell it for INV and deposit into INV market
+/// @dev Require approving DbrHelper to claim on behalf of the user (via setClaimer function in INVEscrow)
+contract DbrHelper is Ownable, ReentrancyGuard {
+    error NoEscrow(address user);
+    error NoDbrToClaim();
+    error AddressZero(address token);
+    error RepayParamsNotCorrect(
+        uint256 percentage,
+        address to,
+        address market,
+        uint256 sellForDola,
+        bool isMarket
+    );
+    error SellPercentageTooHigh();
+    error RepayPercentageTooHigh();
+    error ClaimedWrongAmount(uint256 amount, uint256 claimable);
+
+    IMarket public constant invMarket =
+        IMarket(0xb516247596Ca36bf32876199FBdCaD6B3322330B);
+    ICurvePool public constant curvePool =
+        ICurvePool(0xC7DE47b9Ca2Fc753D6a2F167D8b3e19c6D18b19a);
+    IERC20 public constant dola =
+        IERC20(0x865377367054516e17014CcdED1e7d814EDC9ce4);
+    IERC20 public constant dbr =
+        IERC20(0xAD038Eb671c44b853887A7E32528FaB35dC5D710);
+    IERC20 public constant inv =
+        IERC20(0x41D5D79431A913C4aE7d69a668ecdfE5fF9DFB68);
+
+    uint256 public constant dolaIndex = 0;
+    uint256 public constant dbrIndex = 1;
+    uint256 public constant invIndex = 2;
+    uint256 public constant denominator = 10000; // 100% in basis points
+
+    mapping(address => bool) public isMarket;
+
+    event Sell(
+        address indexed claimer,
+        uint amountIn,
+        uint amountOut,
+        uint indexOut,
+        address indexed receiver
+    );
+    event RepayDebt(
+        address indexed claimer,
+        address indexed market,
+        address indexed to,
+        uint dolaAmount
+    );
+    event DepositInv(
+        address indexed claimer,
+        address indexed to,
+        uint invAmount
+    );
+    event MarketApproved(address indexed market, bool approved);
+
+    constructor() Ownable(msg.sender) {
+        dbr.approve(address(curvePool), type(uint).max);
+        inv.approve(address(invMarket), type(uint).max);
+    }
+
+    struct ClaimAndSell {
+        address toDbr; // Address to receive leftover DBR
+        address toDola; // Address to receive DOLA
+        address toInv; // Address to receive INV deposit
```

Comment:
The idea was to give advanced flexibility since it's not adding extra gas consumption and for example could be useful if someone is using different wallets but has all INV in one so might want to repay another debt or send DBR to another address.